bot/utils/functions.py

```diff
@@ -105,3 +109,23 @@ def format_user_list(users: List[UserProfile], title: str) -> str:
     response = f"📃 {title}:\n"
     response += "```\n" + "\n\n".join(user_lines) + "\n```"
     return response
+
+# async def check_clip_duration_and_permissions(segment: dict, message: Message) -> Optional[tuple]:
+#     start_time = max(0, segment["start"] - settings.EXTEND_BEFORE)
+#     end_time = segment["end"] + settings.EXTEND_AFTER
+#
+#     clip_duration = end_time - start_time
+#     if not await DatabaseManager.is_admin_or_moderator(message.from_user.id) and clip_duration > settings.MAX_CLIP_DURATION:
+#         await message.answer(get_limit_exceeded_clip_duration_message())
+#         return None
+#
+#     return start_time, end_time
+
+async def validate_argument_count(
+    message: Message, min_args: int, reply_invalid_args: Callable[[Message, str], Awaitable[None]], error_message: str,
+) -> bool:
+    content = message.text.split()
+    if len(content) < min_args:
+        await reply_invalid_args(message, error_message)
+        return False
+    return True
```

Comment:
to że przekazujesz callable to idealny dowód na to że powinno to być funkcją klasy bazowej która używa swojej funkcji

Reply:
nieladnie tak resolvowac xd

Reply:
nie no po co to tutaj xD

Reply:
Muszę zobaczyć na PC ale chyba wynieść wyniosłem do klasy bazowej i dałem resolve tylko, że nie usunąłem tej już nieużywanej wersji z function sori xD